readalongs/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.20210825"
+__version__ = "0.1.20210928"
```

Comment:
How did this pass `black --check`? CI should have complained about this file. `black --check readalongs/_versino.py` complains on my machine...

Reply:
In case it's not clear, black wants to add a newline at the end of this file.